cuML/src/dbscan/dbscan.cu

```diff
@@ -26,27 +26,27 @@ using namespace Dbscan;
 
 
 void dbscanFit(const cumlHandle& handle, float *input, int n_rows, int n_cols, float eps, int min_pts,
```

Comment:
A header doc would be nice since this is a key function. On the cython side you have max_elems_per_batch but here it turns to max_elems. Was that intentional to convey something?

Reply:
Not necessarily intentional. The original hardcoded constant was called Max_Elems but as the Python api is closer to the end user, I chose to make that more descriptive. I can make it consistent if you prefer.